### PR TITLE
Add world tour messaging to homepage

### PR DESCRIPTION
**Pull request by chrissnyder**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,6 +35,11 @@
     <div class="home-map" data-hash-association="#!/home" class="active"></div>
 
     <div class="bat-detective app">
+      <div class="world-tour-banner">
+        <h2>Bat Detective World Tour start November 2 with the United Kingdom!</h2>
```
Comment (astopy) [inline index.html]: "start" should be "starts".

Comment (chrissnyder) [inline index.html]: Fixed

File: index.html [hunk 2]
```diff
@@ -35,6 +35,11 @@
     <div class="home-map" data-hash-association="#!/home" class="active"></div>
 
     <div class="bat-detective app">
+      <div class="world-tour-banner">
+        <h2>Bat Detective World Tour starts November 2 with the United Kingdom!</h2>
+        <p>Learn more at the <a href="http://blog.batdetective.org/" target="_blank">blog</a></p>
```
Comment (astopy) [inline index.html]: Presumably the link should now be http://blog.batdetective.org/2015/11/02/bat-detective-world-tour-uk-information/. Still waiting on final wording for the text.

Comment (rorygibb) [inline index.html]: Hi guys, Rory here from Bat Detective. In terms of final wording for the text, I think this should be fine: 

Bat Detective is on a World Tour! We're currently in the United Kingdom. Learn more at the <a href="http://blog.batdetective.org/2015/11/02/bat-detective-world-tour-uk-information/" target="_blank">blog</a>

Where on the front page will this banner be appearing? We also had a few other specific changes it'd be ideal to make to the Bat Detective main site if possible - is it best for me to raise these as separate issues on here? Thanks very much.

**Comment by rorygibb** [comment]
Hi all, we have a few additional changes we'd like to make to the Bat Detective site, if at all possible. I'm listing them as separate comments here...

1) Main banner on front page: 

TEXT: "Bat Detective is on a World Tour! We're currently in the United Kingdom. Learn more at the blog"
LINK TO: http://blog.batdetective.org/2015/11/02/bat-detective-world-tour-uk-information/

**Comment by rorygibb** [comment]
2) Can we change the map on the front page ('powered by CartoDB') so that it is zoomed in over the current World Tour location? Right now we're in the UK so if it could look something like this that'd be ideal --

batdetect_frontpage_shot

**Comment by rorygibb** [comment]
v) Replace the text box on the front page showing “Learn the facts” (see picture below) with the following: 

TITLE: We are on a World Tour!

TEXT: Starting from November 2015 Bat Detective is on a World Tour! We’ll be regularly uploading new data from places across the globe, from Europe to Africa, North America and Asia, each with its own selection of bat species. Read more about where we’re visiting and the science behind Bat Detective here. 

CLICK-BUTTON: “Learn about the World Tour”
LINK TO: http://blog.batdetective.org/2015/11/02/bat-detective-world-tour-uk-information/

learn_facts

**Comment by rorygibb** [comment]
4) Is there anywhere on the main 'Classify' page that we can fit an additional text box letting people know that the data they're classifying is part of the world tour data, and with a link to info about that particular location? Something like:
 
"This data was recorded in the UK. Learn more about bats in the UK here"

**Comment by astopy** [comment]
@rorygibb The main banner text/link will be live shortly. It'll appear on the home page right below the map. The other changes will take a bit more time -- I'm away for the next week now, but I'll take a look at the rest once I'm back.

**Comment by rorygibb** [comment]
Thank you - I see it, looks good. And cheers re. the rest of the changes, just get in touch with me if you have any more specific queries about them when you're back.

**Pull request merged**